### PR TITLE
E325 removal - Part I

### PR DESCRIPTION
**Pull request by Akasurde**
##### SUMMARY
Remove E325 from Amazon modules

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
A bunch of them ...

### CODE REVIEW

**Review by gundalow — APPROVED** [review]

File: lib/ansible/modules/cloud/amazon/ec2_elb_lb.py [hunk 1]
```diff
@@ -24,67 +24,68 @@
...
     description:
-      - List of instance ids to attach to this ELB
+      - List of instance ids to attach to this ELB.
     type: bool
     default: 'no'
     version_added: "2.1"
   purge_instance_ids:
     description:
-      - Purge existing instance ids on ELB that are not found in instance_ids
+      - Purge existing instance ids on ELB that are not found in instance_ids.
```
Comment (willthames) [inline lib/ansible/modules/cloud/amazon/ec2_elb_lb.py]: The changes to add full stops in a subset of modules seems an unrelated concern to this PR. If you want to change punctuation, it might be better to do so in a separate PR

File: lib/ansible/modules/cloud/amazon/ec2_elb_lb.py [hunk 2]
```diff
@@ -24,67 +24,68 @@
...
-      - Purge existing listeners on ELB that are not found in listeners
+      - Purge existing listeners on ELB that are not found in listeners.
     type: bool
     default: 'yes'
   instance_ids:
     description:
-      - List of instance ids to attach to this ELB
+      - List of instance ids to attach to this ELB.
     type: bool
     default: 'no'
```
Comment (willthames) [inline lib/ansible/modules/cloud/amazon/ec2_elb_lb.py]: Do we want consistency of defaults for bools? Some modules use 'no', some modules use False. That might be an unrelated concern too though.

File: lib/ansible/modules/cloud/amazon/ec2_elb_lb.py (hunk 1 above)
Comment (Akasurde) [reply lib/ansible/modules/cloud/amazon/ec2_elb_lb.py]: I removed Punctuation changes. Will address it in a separate PR.

File: lib/ansible/modules/cloud/amazon/ec2_elb_lb.py (hunk 2 above)
Comment (Akasurde) [reply lib/ansible/modules/cloud/amazon/ec2_elb_lb.py]: Type `bool` handles both Booleans and String booleans.

Comment (willthames) [reply lib/ansible/modules/cloud/amazon/ec2_elb_lb.py]: Sure, I understand that, just seems an odd inconsistency in this part of the documentation. But not a blocker.

**Review by willthames — APPROVED** [review]

**Comment by willthames** [comment]
rebuild_merge

File: lib/ansible/modules/cloud/amazon/ec2_elb_lb.py (hunk 2 above)
Comment (gundalow) [reply lib/ansible/modules/cloud/amazon/ec2_elb_lb.py]: `type: bool` expands to yes/no in the module doc generation.

**Comment by Akasurde** [comment]
@gundalow @willthames Thanks for the review.
